### ISSUE TITLE
Add prometheus metrics for sbom report

### ISSUE THREAD

**Issue by chen-keinan** [aquasecurity/trivy-operator#1353]
it is required to add metrics for sbom report

**Comment by rammanoj** [comment]
@chen-keinan, could I know what all metrics would be added as part of this ? and will this be added as part of next release ?

**Comment by chen-keinan** [comment]
Yes its plan for next release , it will include components and dependencies details

**Comment by rammanoj** [comment]
Thanks!

**Comment by MPV** [comment]
What kind of metrics are we thinking of here?

**Comment by chen-keinan** [comment]
@MPV one metric is components and dependencies totals , if you have other in mine feel free to suggest

**Comment by MPV** [comment]
Idea: a counter per license type (spdx is as label)?
